### PR TITLE
Mapped Unlinked Concepts in Kornai-2018-1400

### PR DESCRIPTION
**Pull request by mathildavz**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 1]
```diff
@@ -416,7 +416,7 @@ Kornai-2018-1400-414	414	2555	duty	N			vám	portorium	cło
 Kornai-2018-1400-415	415	1693	each	G	3416	EVERY (EACH)	mind	quisque	kaźdy	all
 Kornai-2018-1400-416	416	1716	eager	A			mohó	avidus	chétny	HAS desire, lack(wait)
 Kornai-2018-1400-417	417	1296	earn	V	733	EARN	keres	mereo	zarabiać	after(=AGT HAS =PAT[money]), before(=AGT[work])
-Kornai-2018-1400-418	418	815	earth	N			föld	terra	ziemia	planet, IN/2758 space/2509, life ON, ocean ON, island ON, HAS atmosphere
+Kornai-2018-1400-418	418	815	earth	N	965	WORLD	föld	terra	ziemia	planet, IN/2758 space/2509, life ON, ocean ON, island ON, HAS atmosphere
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:419]: I'd unmap. WORLD is already mapped to another item.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 2]
```diff
@@ -453,7 +453,7 @@ Kornai-2018-1400-451	451	405	exchange	N	2509	EXCHANGE	csere	mutatio	wymiana	befo
 Kornai-2018-1400-452	452	1167	excite	V			izgat	excito	podniecać	=PAT[person], =AGT CAUSE[=PAT HAS strong (feeling)]
 Kornai-2018-1400-453	453	550	exercise	V			edz	exerceo	ćwiczyć	
 Kornai-2018-1400-454	454	920	exercise	V			gyakorol	exerceo	ćwiczyć	repeat, develop/758, =AGT CAUSE[skill[better]], <sport>
-Kornai-2018-1400-455	455	2587	exist	V			van	exsto	być	real
+Kornai-2018-1400-455	455	2587	exist	V	1579	BE	van	exsto	być	real
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:456]: I'd unmap. BE is already mapped to another item.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 3]
```diff
@@ -562,7 +562,7 @@ Kornai-2018-1400-560	560	839	gradual	A			fokozatos	quod gradatim fit	stopniowy	c
 Kornai-2018-1400-561	561	2183	grain	N	605	GRAIN	szem	granum	ziarno	seed, small, dry, cereal HAS
 Kornai-2018-1400-562	562	877	grass	N	606	GRASS	fű	gramen	trawa	plant, HAS narrow(leaf), COVER ground
 Kornai-2018-1400-563	563		gray	N	153	GREY	szürke	canus	szary	grey
-Kornai-2018-1400-564	564	1746	great	N			nagy	magnus	wielki	big
+Kornai-2018-1400-564	564	1746	great	N	1202	BIG	nagy	magnus	wielki	big
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:565]: I'd unmap. BIG is already mapped.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 4]
```diff
@@ -628,8 +628,8 @@ Kornai-2018-1400-626	626	13	inside	G	1606	INSIDE	-ban	intra	w	in/2758
 Kornai-2018-1400-627	627	1031	instead	D	1652	INSTEAD	helyett	loco	zamiast	lack(=REL), HAS purpose, =REL HAS purpose
 Kornai-2018-1400-628	628	701	instrument	N	1361	TOOL	eszköz	instrumentum	narzédzie	object, work INSTRUMENT, ' USE, HAS purpose, AT/2744 hand
 Kornai-2018-1400-629	629	2143	intend	V			szán	misereor	zamierzać	=PAT[plan], want
-Kornai-2018-1400-630	630	515	interest	N			érdek	commodum	interes	=POSS WANT
-Kornai-2018-1400-631	631	517	interest	N			érdeklődés	studium	zainteresowanie	desire, =POSS WANT[=POSS KNOW]
+Kornai-2018-1400-630	630	515	interest	N	3847	INTEREST (FEELING)	érdek	commodum	interes	=POSS WANT
+Kornai-2018-1400-631	631	517	interest	N	3847	INTEREST (FEELING)	érdeklődés	studium	zainteresowanie	desire, =POSS WANT[=POSS KNOW]
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:632]: I'd unmap this one.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 5]
```diff
@@ -762,7 +762,7 @@ Kornai-2018-1400-760	760	1068	month	N	1370	MONTH	hónap	mensis	miesiác	unit,
 Kornai-2018-1400-761	761	681	moral	N			erkölcs	mos	moralność	
 Kornai-2018-1400-762	762	2306	moral	N			tanulság	documentum	morał	lesson, PART_OF story
 Kornai-2018-1400-763	763	2404	more	A	1992	MORE	több	plus	wiécej	quantity, ER'
-Kornai-2018-1400-764	764	1518	most	A			legtöbb	plurimus	najwiécej	-est
+Kornai-2018-1400-764	764	1518	most	A	1198	MANY	legtöbb	plurimus	najwiécej	-est
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:765]: I'd unmap. MANY is already mapped twice.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 6]
```diff
@@ -783,7 +783,7 @@ Kornai-2018-1400-781	781	2260	necessary	A	1030	NECESSARY	szükséges	necessari
 Kornai-2018-1400-782	782	1803	neck	N	1333	NECK	nyak	cervices	szyja	PART_OF body, CAUSE[head AT/2744]
 Kornai-2018-1400-783	783	2259	need	N	2339	NEED (SOMETHING)	szükség	necessitas	potrzeba	bad FOR/2782 =AGT, lack(=TO) AT =AGT, =AGT WANT lack(=TO)
 Kornai-2018-1400-784	784	2448	needle	N	1382	NEEDLE (FOR SEWING)	tű	acus	igła	artefact, long, thin/2598, steel, pierce, HAS hole, <sew INSTRUMENT>
-Kornai-2018-1400-785	785	3004	negative	A			negatív	negativus	negatywny	lack
+Kornai-2018-1400-785	785	3004	negative	A	1240	NOT	negatív	negativus	negatywny	lack
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:786]: I'd unmap. NOT is already mapped.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 7]
```diff
@@ -914,7 +914,7 @@ Kornai-2018-1400-912	912	1338	port	N	610	PORT	kikötő	portus	port	place/1026,
 Kornai-2018-1400-913	913	1027	position	N			hely	locus	miejsce	place/1026
 Kornai-2018-1400-914	914	297	possess	V			birtokol	possideo	posiadać	=AGT HAS =PAT
 Kornai-2018-1400-915	915	1525	possible	A			lehetséges	possibilis	moźliwy	LET/971, can/1246
-Kornai-2018-1400-916	916	75	post	N			állás	locus/munus	stanowisko	
+Kornai-2018-1400-916	916	75	post	N	281	POST	állás	locus/munus	stanowisko
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:917]: I'd unmap. POST is already mapped.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 8]
```diff
@@ -985,7 +985,7 @@ Kornai-2018-1400-983	983	2658	red	N	156	RED	vörös	ruber	czerwony	colour, war
 Kornai-2018-1400-984	984	2147	regular	A			szabályos	ordinatus	zwykły	CONFORM rule
 Kornai-2018-1400-985	985	2796	relate	V			kapcsolatos			=AGT AT/2744 relation
 Kornai-2018-1400-986	986	3544	relationship	N			viszony	affinitas	zwiázek	
-Kornai-2018-1400-987	987	1496	relax	V			lazít	relaxo	relaksować sié	after(loose)
+Kornai-2018-1400-987	987	1496	relax	V	168	REST	lazít	relaxo	relaksować sié	after(loose)
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:988]: Unmap.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 9]
```diff
@@ -1043,7 +1043,7 @@ Kornai-2018-1400-1041	1041	2350	sea	N	1474	SEA	tenger	mare	morze	ocean
 Kornai-2018-1400-1042	1042	1297	search	V	1468	LOOK FOR	keres	quaero	szukać	effort, SEE place/1026, =AGT WANT[=AGT KNOW place/1026], =PAT AT/2744 place/1026
 Kornai-2018-1400-1043	1043	546	season	N	691	SEASON	évad	tempus	sezon	period[<four>], PART_OF year, spring/2318 IS_A, summer IS_A, fall/1883 IS_A, winter IS_A, HAS weather
 Kornai-2018-1400-1044	1044	548	season	N	691	SEASON	évszak	tempus anni	pora roku	period, time, PART_OF year, activity IN/2758
-Kornai-2018-1400-1045	1045	2494	seat	N			ülés	sedes	siedzenie	sit IN/2758
+Kornai-2018-1400-1045	1045	2494	seat	N	1476	CHAIR	ülés	sedes	siedzenie	sit IN/2758
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1046]: Unmap.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 10]
```diff
@@ -1060,9 +1060,9 @@ Kornai-2018-1400-1058	1058	2951	series	N			sorozat	series	seria	structure, HAS i
 Kornai-2018-1400-1059	1059	1422	serious	A			komoly	gravis	powaźny	deep(feeling), bad, lack(joke), lack(pretend)
 Kornai-2018-1400-1060	1060	2243	serve	V	3648	SERVE (SOMEONE)	szolgál	servio	słuźyć	work, FOR =PAT, <CAUSE[food AT/2744 =PAT]>
 Kornai-2018-1400-1061	1061	2244	service	N			szolgálat	officium	słuźba	=AGT[person, work], work FOR/2782 person[other]
-Kornai-2018-1400-1062	1062	1364	set	V			kitűz	constituo	ustalać	
+Kornai-2018-1400-1062	1062	1364	set	V	160	SET (HEAVENLY BODIES)	kitűz	constituo	ustalać
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1063]: Not sure if the mapping fits. Maybe @martino-vic could help in checking the Hungarian gloss?

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 11]
```diff
@@ -1060,9 +1060,9 @@ Kornai-2018-1400-1058	1058	2951	series	N			sorozat	series	seria	structure, HAS i
 Kornai-2018-1400-1059	1059	1422	serious	A			komoly	gravis	powaźny	deep(feeling), bad, lack(joke), lack(pretend)
 Kornai-2018-1400-1060	1060	2243	serve	V	3648	SERVE (SOMEONE)	szolgál	servio	słuźyć	work, FOR =PAT, <CAUSE[food AT/2744 =PAT]>
 Kornai-2018-1400-1061	1061	2244	service	N			szolgálat	officium	słuźba	=AGT[person, work], work FOR/2782 person[other]
-Kornai-2018-1400-1062	1062	1364	set	V			kitűz	constituo	ustalać	
+Kornai-2018-1400-1062	1062	1364	set	V	160	SET (HEAVENLY BODIES)	kitűz	constituo	ustalać	
 Kornai-2018-1400-1063	1063	2746	set	N			kollekció	classis	kolekcja	group, HAS many(item), together, unit, item HAS common(characteristic)
-Kornai-2018-1400-1064	1064	2375	set	V			tesz	pono	kłaść	=AGT CAUSE[=PAT AT/2744 position[<stable>,<proper>]]
+Kornai-2018-1400-1064	1064	2375	set	V	160	SET (HEAVENLY BODIES)	tesz	pono	kłaść	=AGT CAUSE[=PAT AT/2744 position[<stable>,<proper>]]
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1065]: See comment above.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 12]
```diff
@@ -1108,7 +1108,7 @@ Kornai-2018-1400-1106	1106	163	sleep	U	1585	SLEEP	alszik	dormio	spać	rest, =PA
 Kornai-2018-1400-1107	1107	434	slide	N			csúszik	labor	ślizgać sié	move, ON surface, HAS constant(contact)
 Kornai-2018-1400-1108	1108	396	slight	A			csekély	parvus	niewielki	ER
 Kornai-2018-1400-1109	1109	2091	slippery	A	3702	SLIPPERY	síkos	lubricus	ślizgi	
-Kornai-2018-1400-1110	1110	1529	slope	U			lejt	declivis est	opadać	=AGT HAS direction, HAS end[high], =AGT HAS end[other,low]
+Kornai-2018-1400-1110	1110	1529	slope	U	3076	SLOPE	lejt	declivis est	opadać	=AGT HAS direction, HAS end[high], =AGT HAS end[other,low]
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1111]: Do we know what the "U" stands for?

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 13]
```diff
@@ -1151,13 +1151,13 @@ Kornai-2018-1400-1149	1149	1640	spoil	U			megromlik	corrumpor	zepsuć sié	aft
...
 Kornai-2018-1400-1152	1152	2357	spread	V	38	SPREAD OUT	terít	sterno	szerzyć	
-Kornai-2018-1400-1153	1153	2358	spread	U			terjed	distendo	rozprzestrzeniać sié	after(=PAT AT/2744 wide)
+Kornai-2018-1400-1153	1153	2358	spread	U	38	SPREAD OUT	terjed	distendo	rozprzestrzeniać sié	after(=PAT AT/2744 wide)
 Kornai-2018-1400-1154	1154	1759	square	N	850	SQUARE	négyzet	quadratum	kwadrat	IN/2758 plane/2090, figure/140, HAS side[four, equal]
 Kornai-2018-1400-1155	1155	74	stand	U	1442	STAND	áll	sto	stać	=AGT[upright], =AGT ON two(foot)
 Kornai-2018-1400-1156	1156	1705	standard	N			minta	exemplum	przykład	[compare INSTRUMENT] CAUSE accept, society/2285 HAS, <moral>
 Kornai-2018-1400-1157	1157	2150	standard	A			szabványos	prototypus	standartowy	
 Kornai-2018-1400-1158	1158	408	star	N	1430	STAR	csillag	stella	gwiazda	planet, shine, dot, AT/2744 sky, AT/2744 night
-Kornai-2018-1400-1159	1159	1313	start	N			kezdet	initium	poczátek	after(=POSS)
+Kornai-2018-1400-1159	1159	1313	start	N	3447	BEGIN OR START	kezdet	initium	poczátek	after(=POSS)
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1160]: Unmap. BEGIN OR START is supposed to be the verb meaning.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 14]
```diff
@@ -1243,8 +1243,8 @@ Kornai-2018-1400-1241	1241	1588	then	D	2921	THEN	majd	mox	wtedy	time, follow
 Kornai-2018-1400-1242	1242	707	these	N			ezek	hi/hae/haec	te	this, more
 Kornai-2018-1400-1243	1243	1874	they	G	817	THEY	ők	ii/eae	oni	more(person)
 Kornai-2018-1400-1244	1244	2134	thick	A	1244	THICK	sűrű	densus	gésty	lack(thin/1038)
-Kornai-2018-1400-1245	1245	2752	thick	A			vastag	crassus	gruby	distance[great] BETWEEN surface, HAS more(surface), lack(thin/2598)
-Kornai-2018-1400-1246	1246	1038	thin	A			híg	liquidus	rzadki	[great(distance)] BETWEEN [particle], particle IN/2758, lack(thick/2134)
+Kornai-2018-1400-1245	1245	2752	thick	A	2151	COARSE	vastag	crassus	gruby	distance[great] BETWEEN surface, HAS more(surface), lack(thin/2598)
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1246]: Unmap.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 15]
```diff
@@ -1370,7 +1370,7 @@ Kornai-2018-1400-1368	1368	332	wine	N	1524	WINE	bor	vinum	wino	drink, before(fru
 Kornai-2018-1400-1369	1369	2322	winter	N	1471	WINTER	tél	hiems	zima	season/548, FOLLOW autumn, spring/2318 FOLLOW, cold, snow IN/2758, death IN/2758
 Kornai-2018-1400-1370	1370	485	wire	N	2631	WIRE	drót	filum metallicum	drut	metal, long, thin/2598, <frame>, <electricity IN/2758>
 Kornai-2018-1400-1371	1371	133	wish	V	1784	WANT	akar	volo	chcieć	gentle(=AGT WANT =PAT)
-Kornai-2018-1400-1372	1372	1322	wish	V			kíván	opto	źyczyć	
+Kornai-2018-1400-1372	1372	1322	wish	V	1784	WANT	kíván	opto	źyczyć
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:1373]: Unmap.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 16]
```diff
@@ -57,11 +57,11 @@ Kornai-2018-1400-55	55	2942	Uganda	N			Uganda			country, @Uganda
 Kornai-2018-1400-56	56	3162	Ural	N			Urál			@Ural
 Kornai-2018-1400-57	57	2870	Wales	N			Wales			country, @Wales
 Kornai-2018-1400-58	58	2871	Washington	N			Washington			place/1026, @Washington
-Kornai-2018-1400-59	59	1245	able	A			képes	idoneus	w stanie	can/1246[=AGT[=TO]]
+Kornai-2018-1400-59	59	1245	able	A	972	BE ABLE	képes	idoneus	w stanie	can/1246[=AGT[=TO]]
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Kornai-2018-1400.tsv:60]: Given the POS tag, I'd unmap.

**Review by AnnikaTjuka — COMMENTED** [review]
I left several comments. It's a tricky list. @mathildavz Could you check whether a concept is already mapped in the current version of the list or not? I assume that certain decisions have already been made not to map some glosses and we have to be careful with the mapping here. It would be great if @martino-vic could help us distinguish some of the Hungarian glosses.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv [hunk 17]
```diff
@@ -123,9 +123,9 @@ Kornai-2018-1400-121	121	1253	ask	V	1309	ASK (INQUIRE)	kérdez	quaero	pytać	c
 Kornai-2018-1400-122	122	2744	at	G	1461	AT	-nál	apud	u	=PAT[place/1026]
 Kornai-2018-1400-123	123	39	at	G	1461	AT	-ra	apud	przy	
 Kornai-2018-1400-124	124	181	atom	N			atom	atomus	atom	particle, lack PART_OF, small
-Kornai-2018-1400-125	125	2696	attach	V			édesget	amplexor	przywiázać	bind, =AGT CAUSE love, love IN/2758 =PAT
-Kornai-2018-1400-126	126	1089	attach	V			hozzákapcsol	adiungo	dołáczyć	join
-Kornai-2018-1400-127	127	2034	attack	V			roham	impetus	atak	violent, <INSTRUMENT weapon>, WANT hurt, WANT damage
+Kornai-2018-1400-125	125	2696	attach	V	3894	ATTACH	édesget	amplexor	przywiázać	bind, =AGT CAUSE love, love IN/2758 =PAT
```
[the remark this answers is not in the document]
Comment (martino-vic) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:126]: "édesget" is a very rare word, maybe even archaic. It's the word for "sweet" (édes) with the frequentative suffix "-gat/get". It basically means to "lure" in the sense of "to make someone move towards you with sweet talk". There's a dictionary entry for this [here](https://www.arcanum.com/hu/online-kiadvanyok/Lexikonok-a-magyar-nyelv-ertelmezo-szotara-1BE8B/e-e-2529E/edesget-25401/). The example sentence "Magához édesgette a gyereket." would translate to "They lured the child to them".

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 1 above)
Comment (martino-vic) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:419]: föld can also mean soil. Guess "terra" too, but not sure about that one, not a native Latin speaker lol

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 13 above)
Comment (martino-vic) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:1160]: kezdet is a noun with the -et nominal derivational suffix, the verb is "kezd"

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 16 above)
Comment (martino-vic) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:60]: "képes - be able" is fine

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 17 above)
[the remark this answers is not in the document]
Comment (mathildavz) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:126]: Thank you, @martino-vic! So I would unmap this, @AnnikaTjuka.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 11 above)
Comment (mathildavz) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:1065]: @martino-vic Could you help with this gloss? Do you think it fits?

**Comment by mathildavz** [comment]
Thank you @martino-vic and @AnnikaTjuka for the comments! I have implemented the suggested changes.

**Comment by LinguList** [comment]
I am sorry, I lost track here, but is there anything that should be checked from my side regarding this list, or do you think you have it all covered?

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 10 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:1063]: Given a Google search of the glosses, this needs to be unmapped.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 11 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:1065]: Given a Google search of the glosses, this needs to be unmapped.

File: concepticondata/conceptlists/Kornai-2018-1400.tsv (hunk 12 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Kornai-2018-1400.tsv:1111]: Then this needs to be unmapped.

**Comment by AnnikaTjuka** [comment]
> I am sorry, I lost track here, but is there anything that should be checked from my side regarding this list, or do you think you have it all covered?

@LinguList I think we have it covered. @martino-vic helped differentiate the Hungarian glosses and I'll check the rest of the open comments now.

**Review by AnnikaTjuka — COMMENTED** [review]
A few final comments from my side.

**Comment by mathildavz** [comment]
@AnnikaTjuka Thank you! This should be ready to be merged now.

**Pull request merged**